pages/cloud/private-cloud/interface_secure/guide.fr-fr.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Interface securisée
+slug: interface-secure

```

Comment:
A file with the same slug exists into: https://github.com/simondorval/docs/blob/develop/pages/cloud/private-cloud/interface-secure/guide.fr-fr.md

This causes crashes on static site generation

Reply:
Thanks man :)